### PR TITLE
adds game of life to test files

### PR DESCRIPTION
**Pull request by ablwr**
this is a fun filter

### CODE REVIEW

**Comment by kfrn** [comment]
Whoa awesome!

File: index.html [hunk 1]
```diff
@@ -1880,6 +1880,30 @@ <h3>Makes a broken test file</h3>
...
+        <dt>death_color=#C83232:life_color=#00ff00</dt><dd>specifies color for cell death and cell life; mold_color can also be set</dd>
+        <dt>,</dt><dd>comma signifies closing of video source assertion and ready for filter assertion</dd>
+        <dt>scale=1200:800</dt><dd>scale to 1280 width and 800 height</dd>
+      </dl>
+      <img src="img/life.gif" alt="GIF of above command">
+      <p>To save a portion of the stream instead of playing it back infinitely, use the following command:</p>
+      <p><code>ffmpeg -f lavfi -i life=s=300x200:mold=10:r=60:ratio=0.1:death_color=#C83232:life_color=#00ff00,scale=1200:800 -t 5 <i>output_file</i></code></p>
+      <p class="link"></p>
+    </div>
+    <!-- ends Game of Life -->
```
Comment (retokromer) [inline index.html:1906]: I recommend to add an empty line after the end (for code readability).

**Review by retokromer — APPROVED** [review]
nit, otherwise LGTM
(life is not a game, sadly ;-)

File: index.html (hunk 1 above)
Comment (ablwr) [reply index.html:1906]: for you :)

**Pull request merged**